app/controllers/application_controller.rb

```diff
@@ -261,18 +245,17 @@ def get_stories(user, include_allies)
       my_strategies = ally_strategies
     end
 
-    moments = Moment.where(id: my_moments.map(&:id)).all.order("created_at DESC")
-    strategies = Strategy.where(id: my_strategies.map(&:id)).all.order("created_at DESC")
-
-    if moments.count > 0
-      stories = moments.zip(strategies).flatten.compact
-    else
-      stories = strategies.flatten.compact
-    end
+    moments = Moment.where(id: my_moments.map(&:id)).all.order('created_at DESC')
+    strategies = Strategy.where(id: my_strategies.map(&:id)).all.order('created_at DESC')
 
-    stories = stories.sort_by {|x| x.created_at }.reverse!
+    stories =
```

Comment:
In this case, that would make for a pretty log line and you'll just end up with a new RuboCop error about line length